lib/osf-components/addon/components/addons-service/configured-addon-edit/template.hbs

```diff
@@ -28,104 +28,106 @@
         @startingFolderId={{this.selectedFolder}}
         as |fileManager|
     >
-        <div local-class='current-path'>
-            <Button
-                data-test-go-to-root
-                data-analytics-name='Go to root folder'
-                @layout='fake-link'
-                aria-label={{t 'addons.configure.go-to-root'}}
-                {{on 'click' fileManager.goToRoot}}
-            >
-                <FaIcon @icon='home' />
-                {{t 'general.home'}}
-            </Button>
-            {{#each fileManager.currentPath as |pathItem|}}
+        {{#if this.hasRootFolder }}
```

Comment:
I think this is a bit out of scope for this PR, but I'm wondering if `hasRootFolder` is false, we shouldn't need to invoke the `FileManager` component at all?

Meaning, right now, this template is structured (in pseudo-code) like 
```
<FileManager>
   {{#if hasRootFolder}}
      ...some file explorer stuff
      <div>
         <Button>Save</Button>
         <Button>Cancel</Button>
      </div>
   {{/if}}
</FileManager>
```

But I think we can avoid a few unneeded (and possibly prone to fail) network calls by structuring it like:
```
{{#if hasRootFolder}}
   <FileManager>
      ...some file explorer stuff

   </FileManager>
{{/if}}
<div>
   <Button>Save</Button>
   <Button>Cancel</Button>
</div>
```

I don't want to scope-creep this PR, so I can make these changes in a separate PR if you would prefer!

Reply:
Lemme try it real quick, see how it goes.

Reply:
"real quick" == 3 hours apparently